AODN/AODN-WAVE-NRT/ARDC_API_NRT/ardc_nrt/config/sofar/sources_id_metadata.json

```diff
@@ -204,7 +204,7 @@
    "SPOT-0317":
   {
     "spotter_id": "SPOT-0317",
-    "site_name": "Inverloch Inshore",
+    "site_name": "Inverloch Inshore - unused",
```

Comment:
if it's unused, why not removing it from the list of spotters? this will create a folder of data with unused in it. pretty ugl

Reply:
Kind of on purpose. 
Currently some of the deployments have no start date and that's how it is distinguished from current valid ones, keeoing the buoy in case it gets redeployed (instead of reentering all the infos). 
With the new code (and arguably I should have waited to have this in the PR) if "unused" is in the name, it just skips it. 
The idea is also not to lose track of deployed buoys that just have been moved, because I can see the amount that are missing/wrong coordinates atm and it's going to be fun to figure out the deployment start and end if they are removed every time...
I personally think that it would be better to have the old deployments in a separate file that can be referred to if needs be, but until I do that, I prefer to keep them in the config file.

Reply:
I think having unused in the config like this , to trigger something in the code is very hacky (and ugly). Far to easy to froget about it.

I'd rather see a new key in the configuration such as "publish" : True/False . and having the code automatically assume it's true if the key is missing. And if the key is set in the config, well use it. And if it's False, don't publish it.

Reply:
fair enough, I will find a different way

Reply:
Removed 4 spotters unused or without deployment dates